packages/top-app-bar/index.tsx

```diff
@@ -19,28 +19,60 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
-
-import React from 'react';
-import PropTypes from 'prop-types';
-import classnames from 'classnames';
-import TopAppBarFixedAdjust from './FixedAdjust';
+import * as React from 'react';
+import * as classnames from 'classnames';
+import TopAppBarFixedAdjust, {TopAppbarFixedAdjustProps} from './FixedAdjust';
 import {
   MDCFixedTopAppBarFoundation,
   MDCTopAppBarFoundation,
   MDCShortTopAppBarFoundation,
+// no mdc .d.ts file
+// @ts-ignore
 } from '@material/top-app-bar/dist/mdc.topAppBar';
 
-export default class TopAppBar extends React.Component {
-  foundation_ = null;
+export interface TopAppBarProps {
+  actionItems?: React.ReactElement<any>[];
```

Comment:
Usually any is not allowed when using --noImplicitAny, which google3 uses. Is there any reason the generic type cannot be specified ahead of time?

Reply:
I think --noImplicitAny will allow you to imply any types, but defining it as `any` is ok. However it is the lazy way out. I think in this case, it'll be an <i> or <svg> or <img>.